Gemfile

```diff
@@ -28,6 +28,7 @@ gem "rails", "~> 6.1.6", ">= 6.1.6.1"
 gem "rails-i18n"
 gem "ransack"
 gem "sass-rails", ">= 6"
+gem "sidekiq"
```

Comment:
thêm gem redis vào nhé
Tìm hiểu xem gem này dùng để làm gì?

Reply:
Đây là nơi lưu trữ các jobs khi sử dụng Sidekiq. Là một in-memory database và cũng có thể coi như một NoSQL database. Lưu trữ dữ liệu trong RAM nên có tốc độ nhanh. Nó giữ tất cả dữ liệu job trong thời gian thực và lịch sử của dữ liệu trong Sidekiq's Web.